src/linagora.esn.calendar/app/components/calendar-color-picker/calendar-color-picker.js

```diff
@@ -67,9 +67,9 @@
 
       $modal({
         template: require("./calendar-color-picker.pug"),
-        controller: function($scope) {
+        controller: ['$scope',function($scope) {
```

Comment:
We do not use syntax like this. Instead of, we use `/* @ngInject */`

Check this PR for example: https://github.com/OpenPaaS-Suite/esn-frontend-inbox/pull/35/files